css/style.css

```diff
@@ -47,31 +47,137 @@ Developer @GA in NYC
 
 body {
      color: #000;
-     font-size: 12px;
      line-height: 1.4;
-     font-family: Helvetica, Arial, sans-serif;
-     background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
-      background-size: cover;
+     font-family: 'Open Sans', sans-serif;
 }
 
-
 /******************************************
 /* LAYOUT
 /*******************************************/
 
 /* Center the container */
-.container {
-  width: 960px;
-  margin: auto;
-  height: 500px;
+#container {
+  width: 1024px;
+  margin: 0 auto;
 }
 
-header {
+.header {
+  background: url("../images/header_bg.jpg") no-repeat center top rgb(134,141,157);
+  background-size: cover;
+  height: 595px;
+}
 
+.relaxr {
+  font-weight: extra bold;
+  font-size: 35px;
+  color: white;
 }
 
-footer {
+#title {
+  margin-left: 85px;
+  padding-top: 25px;
+}
+
+.tap {
+  font-size: 30px;
+  font-weight: semibold;
+  text-align: center;
+  color: white;
+  margin-top: 100px;
+}
+
+.get {
+  font-weight: extra bold;
+  font-size: 18px;
+  border: 5px;
+  background-color: #f9e42e;
+  padding: 10px 45px 10px 45px;
+  margin-left: 405px;
+  margin-top: 45px;
+  border-radius: 5px;
+}
+
+#benefits_div {
+  background-color: #f0efef;
+  padding: 50px 500px 50px 100px;
+}
+
+.benefits {
+  font-weight: semibold;
+  font-size: 25px;
+  color: #033048;
+}
+
+.enjoy {
+  font-weight: lighter;
+  font-size: 20px;
+  color: #606161;
+}
+
+.testimonial {
+  background: url("../images/testimonial_bg.jpg") no-repeat center;
+  background-size: cover;
+  height: 300px;
+  padding-top: 5px;
+  text-align: center;
+  color: white;
+}
 
+.quote {
+  font-style: italic;
+  font-weight: lighter;
+  font-size: 25px;
+  padding: 20px 200px 0px 200px;
+  line-height: 40px;
+}
+
+.amanda {
+  font-weight: semibold;
+  font-size: 25px;
+  margin-top: 0px;
+  margin-bottom: 0px;
+}
+
+div.signup {
+  background-color: #f9e42e;
+  height: 100px;
+  text-align: center;
+}
+
+#signup_button {
+  font-weight: extra bold;
+  font-size: 18px;
+  border: 5px;
+  border-radius: 4px;
+  background-color: #033048;
+  color: white;
+  margin-top: 30px;
+  padding: 10px 60px 10px 60px;
```

Comment:
@eloisebarrow this could also be written as `padding`: `10px 60px` were the first value corresponds to `top` & `bottom` and the second value corresponds to `left' & `right`